src/cmd_context/cmd_context.h

```diff
@@ -295,6 +299,10 @@ class cmd_context : public progress_callback, public tactic_manager, public ast_
 
     stopwatch                    m_watch;
 
+    smt2::parser*                m_parser;
```

Comment:
seems this would be a property of the API (api_context) not cmd_context

Reply:
Moved to api_context.